.semaphore/semaphore.yml

```diff
@@ -100,6 +100,23 @@ blocks:
       commands:
       - make ci-preflight-checks
 
+- name: "FOSSA scan"
+  run:
+    when: "branch =~ '^release-v' OR branch = 'master'"
+  execution_time_limit:
+    minutes: 30
+  dependencies: ["Prerequisites"]
+  task:
+    prologue:
+      commands:
+      - "curl -H 'Cache-Control: no-cache' https://raw.githubusercontent.com/fossas/fossa-cli/master/install-latest.sh | bash"
```

Comment:
might be a good idea to add the `-fsSL` flags to `curl` here

Reply:
```suggestion
      - "curl -fsSLH 'Cache-Control: no-cache' https://raw.githubusercontent.com/fossas/fossa-cli/master/install-latest.sh | bash"
```